pkg/disttask/framework/storage/task_table.go

```diff
@@ -530,8 +530,8 @@ func (mgr *TaskManager) GetFirstSubtaskInStates(ctx context.Context, tidbID stri
 	return Row2SubTask(rs[0]), nil
 }
 
-// UpdateErrorToSubtask updates the error to subtask.
-func (mgr *TaskManager) UpdateErrorToSubtask(ctx context.Context, execID string, taskID int64, err error) error {
+// FailedSubtask update the task's subtask state to failed and set the err.
+func (mgr *TaskManager) FailedSubtask(ctx context.Context, execID string, taskID int64, err error) error {
```

Comment:
Failed -> Fail? The present tense is better.

Reply:
> Failed -> Fail? The present tense is better.

Followed task related naming, "Failed" represent the expected state after the function

Reply:
Done